### ISSUE TITLE
As a developer i need to set "Packager" and "Group" tags in RPM

### ISSUE THREAD

**Issue by mcbialy** [jordansissel/fpm#1277]
Hello!

In default these tags are filled by default (for Group) or username@hostname (for Packager)
Actually i have to edit them manually.
Could you add an option which makes it easier to change these tags?

**Comment by apiemont** [comment]
you can set packager via command line with 

`$ fpm .... -m "Joe Doe <jdoe@nowhere.com>" ...`

same for groups

`$ fpm .... --rpm-group "Some group name" ...`

then, in general you can also add the -e option to open the spec file and adjust other fields which are not directly accessible via CLI

**Comment by mcbialy** [comment]
Yes, but i want to use it automatically in some script, not via '-e' option.

**Comment by jordansissel** [comment]
Maciej, can you reread the previous comment? The '-e' suggestion was the
end of the comment and before it was the answer to your question.

**Comment by bond-os** [comment]
A note for future readers:
* `--rpm-group` is for setting default filesystem group for package contents (`--rpm-group agroup` results in `%defattr(-,root,agroup,-)` in spec file)
* To set the rpm tag `Group`, you need to use `--category` parameter
